159356-1702-Extramural/capstone
# Feature/largest army

To test:

Use {url:3000}/?setup=skip&startWithCards=20

Test player receives largest army at start of following turn, and can lose it to an opponent.
Test two players eligible on same turn, neither gets it.
Check scores update as army changes hands.
